operations/service_web.go

```diff
@@ -75,6 +76,19 @@ func startWebService() cli.Command {
 			}
 			uiServer := service.GetServer(settings.Ui.HttpListenAddr, uiHandler)
 
+			newRelic, err := settings.SetUpNewRelic()
```

Comment:
should we also do this for the API server? why don't we do it in the service package rather than doing it here?

Reply:
This is for both, I think. The router, from 61, is passed to both the apiHandler and the uiHandler. We have to do it here because it has to happen *after* we attach routes.